shelley/chain-and-ledger/formal-spec/Properties.md

```diff
@@ -1,3 +1,34 @@
+# Ledger and Epoch State Validity
+
+We only care that the properties below are satisfied for _valid_ ledger states, and
+more generally, valid _epoch_ states. Checking things for invalid states should
+not be performed. For this reason, we must have boolean functions that
+evaluate to `True` for valid states. As the STS rule system we have defined
+is deterministic, all valid states can be reached using the transitions in the system,
+and the only states that are valid are those that can be described by a sequence
+of rule applications. Thus, a high-level definition of such a validity function
+would be:
+
+* `validState(s) = True` when `s` is either a valid initial state, or there is
+some sequence of transition rules that can be applied, starting from an
```

Comment:
sequence of signals applied by a given rule?

Reply:
should this be `starting from a valid initial state` ?